config/initializers/devise.rb

```diff
@@ -125,7 +125,9 @@
   # able to access the website for two days without confirming their account,
   # access will be blocked just in the third day. Default is 0.days, meaning
   # the user cannot access the website without confirming their account.
-  # config.allow_unconfirmed_access_for = 2.days
+
+  # a value bigger than 0 is required here for all our tests to work
+  config.allow_unconfirmed_access_for = 1.day
```

Comment:
Gets the user sufficient information about that time limit?

Reply:
Interesting question :D

I just tested it: You do not see this time limit anywhere. But you immediately get the account confirmation mail. When you try to sign in after the day has passed, you get the error message "Du musst Deinen Account bestätigen, bevor Du fortfahren kannst."

Is this enough?

Reply:
Hmm, I think, it is more convenient, if the error message is something like "Der genutzte Link ist leider abgelaufen. Fordere einen neuen an."

Reply:
This is not the amount of time you have to finally confirm your account. This is the amount of time you can use your account without confirming.

Reply:
Then it is ok